lib/auth/cognito.ts

```diff
@@ -117,6 +117,20 @@ export const begin2FAAuthentication = async ({
   email,
   token,
 }: CognitoToken): Promise<AuthenticationFlowToken> => {
+  // ensure the user account is active
+  const prismaUser = await prisma.user.findUnique({
+    where: {
+      email: email,
```

Comment:
Should probably use the sanitizedEmail variable (declared immiediatley after) for consistency.